internal/sql/validate/cmd.go

```diff
@@ -45,6 +47,11 @@ func validateCopyfrom(n ast.Node) error {
 }
 
 func validateBatch(n ast.Node) error {
+	namedFunc := astutils.Search(n, named.IsParamFunc)
+	namedSign := astutils.Search(n, named.IsParamSign)
+	if len(namedFunc.Items)+len(namedSign.Items) > 0 {
+		return nil
+	}
```

Comment:
As some point, we rewrite the AST to change named parameters into normal parameter references. We probably just want to move the validate batch call so that it happens after that process. Then it doesn't need to know anything about named parameters.